src/account/Account.ts

```diff
@@ -183,4 +184,38 @@ export class Account implements IAccount {
       );
     });
   }
+
+  /**
+   * This method aims to interact with a smart contract deployed on the massa blockchain on behalf of the sender.
+   *
+   * @param contractAddress - The address of the smart contract.
+   * @param functionName - The name of the function to be called.
+   * @param parameter - The parameters of the function to be called (array composed of string, bigint and/or boolean).
+   * @param fee - The fee to be paid for the transaction execution by the node.
+   * @returns An ITransactionDetails object. It contains the operationId on the network.
+   *
+   */
+  public async interactWithSC(
+    contractAddress: string,
+    functionName: string,
+    parameter: (string | bigint | boolean)[],
+    fee: bigint,
+  ): Promise<ITransactionDetails> {
+    return new Promise<ITransactionDetails>((resolve, reject) => {
+      connector.sendMessageToContentScript(
+        this._providerName,
+        AvailableCommands.AccountInteractWithSC,
+        {
+          contractAddress,
+          functionName,
+          parameter,
+          fee,
```

Comment:
I don't think you can pass a bigInt to the extension: this object is not JSON serailizable. You should chnage its type here to a string